viite-UI/src/view/ProjectEditForm.js

```diff
@@ -462,9 +464,32 @@
       });
 
       rootElement.on('click', '.project-form button.send', function(){
-        projectCollection.publishProject();
+        new GenericConfirmPopup("Haluatko varmasti poistaa tämän projektin?", {
+          successCallback: function () {
+            //projectCollection.publishProject();
```

Comment:
Remove if not needed please.

Reply:
This one is essencial :smile: removing the comment